resume/README.md

```diff
@@ -0,0 +1,49 @@
+
+# Manpreet Kaur . Jarvis Consulting
+
+I recently graduated from Humber College where I completed a Post-Graduate Certificate in Information Technology Solutions. Prior to pursuing a career in information technology, I was able to gain work experience as an I.T. Process Consultant where I utilized my strengths and skillsets in service management. This professional experience empowered me to pursue a career in information technology solutions. As a result, my career path has led me to pursue an opportunity and work as an Associate Data Engineer at Jarvis Consulting. Additionally, when I'm not working, I am also passionate about music, building software and all things vegetarian food. I also love playing sports, especially volleyball which enables me to utilize my passion for social interactions and teamwork. Going forward, I aim to enhance my skills as IT professional and grow my network.
```

Comment:
Please run a spelling and grammar check using https://www.grammarly.com/ if necessary